cpu/stm32f3/include/cpu_conf.h

```diff
@@ -43,6 +43,20 @@ extern "C" {
 #define CPU_FLASH_BASE                  FLASH_BASE
 /** @} */
 
+/**
+ * @name    Flash page configuration
+ * @{
+ */
+#define FLASHPAGE_SIZE      (2048U)
+#define FLASHPAGE_NUMOF     (STM32_FLASHSIZE / FLASHPAGE_SIZE)
+
+/* The minimum block size which can be written is 2B. However, the erase
+ * block is always FLASHPAGE_SIZE.
+ */
+#define FLASHPAGE_RAW_BLOCKSIZE    (2U)
+/* Writing should be always 4 bytes aligned */
+#define FLASHPAGE_RAW_ALIGNMENT    (4U)
```

Comment:
Why 4 bytes? The blocksize is 2 bytes, skimming through the datasheet I couldn't find a reference to this being a requirement.

Reply:
I found the answer to this in a comment in `tests/periph_flashpage`:

```
The flash page in RAM must be correctly aligned, even in RAM, when
 *          using flashpage_raw. This is because some architecture uses
 *          32 bit alignment implicitly and there are cases (stm32l4) that
 *          requires 64 bit alignment.
```

So its an architecture requirement, and does need to be 32b aligned.

Reply:
Thanks @fjmolinas :)